### ISSUE TITLE
UI3: Device theme makes login badly visible

### ISSUE THREAD

**Issue by WebSpider** [filesender/filesender#2013]
When using the new theme selector added in https://github.com/filesender/filesender/pull/1841 and setting the device preference to 'Dark', the background and font colours are so similar, the landing page is hardly visible.

You can press the login button, but hardly read any text

![image](https://github.com/user-attachments/assets/d02bb84f-86bb-4634-a0ce-cf998797b4ce)

This problem exists on:

- Login page
- Logout page
- All admin pages

**Comment by monkeyiq** [comment]
I find that in Chrome if I select the theme it only applies on that page. If I navigate to another page it drops back to the default theme. If you are able to select a theme I should put some more time in to why theme selection isn't working here. That is also likely to happen for others.

**Comment by WebSpider** [comment]
I dont think i can select a theme without being logged in, but will investigate

**Comment by monkeyiq** [comment]
I can only select the theme on the my profile page. If I leave that page the theme is forgotten.

**Comment by WebSpider** [comment]
Yes, I cannot select the theme if I'm not logged in, and with a dark device preference, the frontpage looks wonky.

**Comment by monkeyiq** [comment]
I guess this might change with the potential introduction of https://github.com/filesender/filesender/pull/2054

**Comment by monkeyiq** [comment]
I also can not see the themes in PR 2054. So I guess I rely on others to let me know how this works :(
